ci/debian/centreon-web.postinst

```diff
@@ -17,7 +17,9 @@ if [ "$1" = "configure" ] ; then
     chmod -R 0775 \
       /usr/share/centreon/www \
       /usr/share/centreon/GPL_LIB/SmartyCache
-  
+    chmod -R 0664 \
```

Comment:
```suggestion
    chmod 0664 \
```
as they are files, you don't need the parameter `-R`

Reply:
Yep thats what happens when you copy / paste like a pig 💯